duopull/duopull.go

```diff
@@ -293,15 +294,18 @@ func (e *emitter) emit() error {
 	for _, v := range e.events {
 		cv, err := v.toInterface()
 		if err != nil {
-			return err
+			log.Errorf("Can't convert to interface: %s", err)
```

Comment:
Do you think it is worth logging the raw event that caused the error in each of these cases so we can easily track it down?

Reply:
Yeah absolutely, will fix.